doc/sphinx-guides/source/_static/installation/files/etc/shibboleth/shibboleth2_adfs.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<!--
+example shibboleth2.xml for ADFS 3.0
+
+- attribute-map_adfs.xml
+-->
+<SPConfig xmlns="urn:mace:shibboleth:3.0:native:sp:config" xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion" clockSkew="1800">
+  <!-- adfs libraries for shibboleth and apache -->
+  <OutOfProcess>
+    <Extensions>
+      <Library path="adfs.so" fatal="true"/>
+    </Extensions>
+  </OutOfProcess>
+  <InProcess>
+    <Extensions>
+      <Library path="adfs-lite.so" fatal="true"/>
```

Comment:
How does one install these .so files? Do they come in the shib RPM?

Reply:
Probably; will confirm.

Reply:
@pameyer thanks, but no need. I found this in the docs that says these files do come from the shib RPM:

"The WS-Federation code for the SP is supplied in an extension that is included with the SP source code, and is built by default and included with binary packages. The extension is contained in two libraries, one for shibd and one for the web server filter/module. These are named adfs.so and adfs-lite.so respectively." https://wiki.shibboleth.net/confluence/display/SP3/ADFS

Reply:
I did nothing special to UNC's Shib to allow ADFS to work with it... just wedged in the custom IDP XML.

Reply:
I didn't try with the adfs.so/adfs-lite.so removed; but it's a definite possibility that they aren't needed.

Reply:
Great work here.

Reply:
(was testing something, but great work, for real :))